osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -97,16 +100,29 @@ private void updateMods()
                 ranked &= mod.Ranked;
             }
 
-            MultiplierLabel.Text = $"{multiplier:N2}x";
-            if (!ranked)
-                MultiplierLabel.Text += " (Unranked)";
+            ScoreLabel.Text = "Score Multiplier:";

```

Comment:
This doesn't need to be set with each update. As it's static text at this point you can just set it at label declaration.